pages/common/docker-swarm.md

```diff
@@ -0,0 +1,32 @@
+# docker swarm
+
+> A container orchestration tool.
+> More information: <https://docs.docker.com/engine/swarm/>.
+
+- Initialize a swarm cluster:
+
+`docker swarm init`
+
+- Get join token for master or worker:
+
+`docker swarm join-token {{type}}`
+
+- Join new node to cluster:
+
+`docker swarm join --token {{token}} {{manager_node_url:2377}}`
+
+- Leave the cluster:
+
+`docker swarm leave`
+
+- View current CA certificate:
```

Comment:
Based on the docs:
```suggestion
- Display current CA certificate in PEM format:
```
The docs talk about a `current root CA` but I think `root` make some noise here.